server/routes/testutils/appSetup.ts

```diff
@@ -9,12 +9,19 @@ import * as auth from '../../authentication/auth'
 import type { Services } from '../../services'
 import type { ApplicationInfo } from '../../applicationInfo'
 
-const testAppInfo: ApplicationInfo = {
-  applicationName: 'test',
-  buildNumber: '1',
-  gitRef: 'long ref',
-  gitShortHash: 'short ref',
-}
+jest.mock('../../applicationInfo.ts', () => {
+  return {
+    __esModule: true,
+    default: jest.fn(() => {
+      return {
+        applicationName: 'test',
+        buildNumber: '1',
+        gitRef: 'long ref',
+        gitShortHash: 'short ref',
+      } as ApplicationInfo
+    }),
+  }
+})
```

Comment:
mmmh... why can't you inject the application info into the place where you're using it?
 
you should be able to pass it in along with the service overrides [here](https://github.com/ministryofjustice/hmpps-template-typescript/pull/201/files#diff-47e00f44a4610e3f7bcaad8433c7bccf2ae050a627a3399de9553d7c910cd7b0R68)

Reply:
Have you got an example of where it was causing you an issue?

Reply:
I'm sure I tried that but didn't seem to work. I'll give it another go. Just saves having to do that in every test file, and the build info is never going to want pulling from `package.json` in tests?

Reply:
@andrewrlee in this PR (it's actually Draft), when trying to mock out the routes I think.... https://github.com/ministryofjustice/hmpps-developer-portal/pull/2/files